### ISSUE TITLE
Bounced back to Summary tab when viewing my plugin's tab

### ISSUE THREAD

**Issue by ImJasonH** [vmware-tanzu/octant#295]
I added a tab for a CRD type:

```
func handleTabPrint(request *service.PrintRequest) (plugin.TabResponse, error) {
  // ...
  gv, err := pipelineToGraphviz(&p)
  if err != nil {
    return plugin.TabResponse{}, err
  }
  fl := component.NewFlexLayout("Visualization")
  fl.AddSections([]component.FlexLayoutItem{{
    Width: component.WidthFull,
    View:  component.NewGraphviz(gv),
  }})
  return plugin.TabResponse{
    Tab: component.NewTabWithContents(*fl),
  }, nil
}
```

After v0.7.0 (thanks by the way! :tada:) this renders the graphviz as expected (again, woohoo! :tada:) but now any time I click on the tab I created, about 2-3 seconds later the view bounces back to the Summary tab, without any interaction on my part.

Here's a potato-quality gif screencap:
![capture](https://user-images.githubusercontent.com/210737/65175189-63390280-da20-11e9-9d62-62f85f646df4.gif)

In the Chrome console:
```
No script tag of type "javascript/worker" was found and "useWorker" is true. Not using web worker.
Invalid asm.js: Undefined global variable
```
![Screenshot from 2019-09-18 14-31-00](https://user-images.githubusercontent.com/210737/65175488-f5410b00-da20-11e9-9b26-4486a2ed4859.png)

```
$ ~/Downloads/octant_0.7.0_Linux-64bit/octant version
Version:  0.7.0
Git commit:  632f678f1706ad6ce3e725e30f3d267d7231b136
Built:  2019-09-18T16:40:12Z
```

---

The 0.7.0 release seems to mention this bug, but links to a different bug (#240):

```
Fixed bug where tabs generated from plugins would switch back to summary (#240, @GuessWhoSamFoo)
```

cc @GuessWhoSamFoo @wwitzel3 

(And thanks again for this awesome project and for fixing all the bugs so far, I'm _unreasonably_ excited to see where this goes, and what I can do with it in the future :1st_place_medal:)

**Comment by ImJasonH** [comment]
It looks like https://github.com/vmware/octant/issues/181 is another report of this bug, which claims to be fixed. It just doesn't seem to be in 0.7.0, unless I'm missing something. :thinking:

**Comment by wwitzel3** [comment]
@ImJasonH thanks, I'll take another look at this and be sure this gets fixed in 0.7.1

**Comment by GuessWhoSamFoo** [comment]
The changelog should have linked to #222. This behavior is not observed with the sample plugin. Wonder if it is something to do with the graphvis component.

**Comment by wwitzel3** [comment]
@GuessWhoSamFoo yeah, I noticed the same thing, the sample plugin is fine. I think investigating if this is specific to the grapviz component is a good start.

**Comment by ImJasonH** [comment]
I'm unable to reproduce this with the sample plugin, even changing line 76 to `contents := components.NewGraphviz("...some example graphviz I found on the internet...")`

I _am_ able to reproduce with my plugin printing the same example graphviz.

It might be an issue with tabs for CRD types specifically, since the sample plugin adds a tab for a built-in Pod type? :man_shrugging:

**Comment by wwitzel3** [comment]
Thank you for the extra details. That helps.

**Comment by wwitzel3** [comment]
I think this was fixed by #369 
I haven't verified this yet, but will first thing tomorrow.

**Comment by wwitzel3** [comment]
Fixed in master by #369
